index.bs

```diff
@@ -642,6 +642,7 @@ When this method is invoked, the user agent MUST run the following steps:
 
   1. Let |session| be the target {{XRSession}}.
   1. If |session|'s [=ended=] value is <code>true</code>, throw an {{InvalidStateError}} and abort these steps.
+  1. If |newState|'s {{XRRenderStateInit/layers}}'s value is not <code>null</code>, throw an {{InvalidStateError}} and abort these steps.
```

Comment:
I was suggesting that you have something like

```
1. If |newState|'s {{XRRenderStateInit/layers}}'s value is not <code>null</code>If |newState|'s {{XRRenderStateInit/layers}}'s value is not <code>null</code>, run [=update layer state=] with [=pending render state=]
```

and put it right before the `depthNear` line

and in the definition for that algorithm, just throw an InvalidStateError with a note that the layers spec will introduce new semantics.

Reply:
OK. take a look and let me know what you think.